internal/dataplane/parser/translate_kong.go

```diff
@@ -0,0 +1,170 @@
+package parser
```

Comment:
`translate_kong_layer4.go` or separate TCPIngress/UDPIngress files would be clearer about what this file contains.

Reply:
sounds good, done in 3364f3c055a60c0091da31f29b742761e082f236